srcalc/src/main/java/gov/va/med/srcalc/domain/model/RiskModel.java

```diff
@@ -17,6 +18,7 @@
  * A risk model definition.
  */
 @Entity
+@Table(name = "risk_model")
```

Comment:
This attribute is unnecessary and having it is inconsistent with other entities. Please drop it.